dojo/jira_link/helper.py

```diff
@@ -984,7 +984,16 @@ def push_status_to_jira(obj, jira_instance, jira, issue, save=False):
 
 # gets the metadata for the default issue type in this jira project
 def get_jira_meta(jira, jira_project):
-    meta = jira.createmeta(projectKeys=jira_project.project_key, issuetypeNames=jira_project.jira_instance.default_issue_type, expand="projects.issuetypes.fields")
+    meta = None
+    try:
+        meta = jira.createmeta(projectKeys=jira_project.project_key, issuetypeNames=jira_project.jira_instance.default_issue_type, expand="projects.issuetypes.fields")
+    except JIRAError as e:
+        message = "error while calling jira.createmeta(). status: %d, message: %s" % (e.status_code, e.text)
```

Comment:
```suggestion
        message = "Invalid JIRA Project Config, can't retrieve or create metadata. status: %d, message: %s" % (e.status_code, e.text)
```

I do not think it is a good idea to disclose the function name to the end user